tweakwcs/__init__.py

```diff
@@ -7,12 +7,12 @@
 __taskname__ = 'tweakwcs'
 __author__ = 'Mihai Cara'
 
-from pkg_resources import get_distribution, DistributionNotFound
-try:
-    __version__ = get_distribution(__name__).version
-except DistributionNotFound:
-    # package is not installed
-    __version__ = 'UNKNOWN'
+
+from importlib.metadata import version
```

Comment:
I don't think you need importlib at all. Try this:

```
try:
    from ._version import version as __version__
except ImportError:
    __version__ = ''
```

Reply:
I would also include `ModuleNotFoundError`:
```python
try:
    from ._version import version as __version__
except (ImportError, ModuleNotFoundError):
    __version__ = ''```

Reply:
No one should blindly copy-paste stuff but point taken and it doesn't hurt.